static/css/style.css

```diff
@@ -34,6 +34,17 @@ body {
 		padding-right: 20px;
 	}
 }
+.CClogo{
+	height: 40px;
+	width: 40px;
+	position: relative;
+	margin: 0px;
+	margin-right: 30px;
+	margin-left: 10px;
```

Comment:
noted, but at this point it is easier for my understanding to have all. Must I correct this?